test/verify/check-machines-dbus

```diff
@@ -188,6 +188,73 @@ class TestMachinesDBus(machineslib.TestMachines):
         b.wait_not_in_text("#vm-subVmTest1-boot-order", bootOrder)
         b.wait_not_present("#boot-order-tooltip")
 
+    def testDomainMemorySettings(self):
+        b = self.browser
+
+        args = self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-overview") # wait for the tab
+        b.click("#vm-subVmTest1-overview") # open the "Overview" subtab
+
+        # Wait for the edit link
+        b.wait_present("#vm-subVmTest1-memory-count")
+        b.click("#vm-subVmTest1-memory-count")
+
+        # Change memory
+        b.wait_present("#vm-subVmTest1-memory-modal-memory")
+        # VM initially should have 256MiB
+        current_memory = int(b.attr("#vm-subVmTest1-memory-modal-memory", "value"))
```

Comment:
Can this assumption (that it's 256 MiB) be tested here? To make sure that the initial value is correct? The VM creation has full control, so it should be predictable.

Reply:
Done